pkg/runtime/runtime.go

```diff
@@ -478,6 +484,19 @@ func (a *DaprRuntime) initRuntime(opts *runtimeOpts) error {
 		log.Warnf("failed to load components: %s", err)
 	}
 
+	// Enable dynamic loading in standalone mode if dynamic loading preview feature is enabled and components path is provided
+	if a.runtimeConfig.Mode == modes.StandaloneMode && a.runtimeConfig.Standalone.EnableDynamicLoading {
```

Comment:
I would lean towards saying that this block should be moved at the very end of the initialization of the runtime. So we don't risk interfering with the loading of "static" components. WDYT?